sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/OrderByDocumentQueryExecutionContext.java

```diff
@@ -341,7 +341,7 @@ private void initializeWithTokenAndFilter(Map<FeedRangeEpkImpl, OrderByContinuat
                 targetRange,
                 feedRange,
                 collectionRid,
-                () -> client.getResetSessionTokenRetryPolicy().getRequestPolicy(),
```

Comment:
@mbhaskar  please check this, this is not related change , but found we were duplicating this code

Reply:
Good catch. This is redundant as the caller is doing the same.